### PR TITLE
feat(axios-extension): add support for Abap cds view service generation

### PR DESCRIPTION
**Pull request by docirl**
https://github.com/SAP/open-ux-tools/issues/2154

Add support for abap cds views to the Ui service generator in axios extension

### CODE REVIEW

File: examples/odata-cli/src/activities.ts [hunk 1]
```diff
@@ -210,6 +211,10 @@ export async function testUiServiceGenerator(
     const bo = bos.find((bo) => bo.name === env.TEST_BO_NAME);
     logger.debug(bos.map((bo) => bo.name));
 
+    // Get Abap CDS Views
```
Comment (longieirl) [inline examples/odata-cli/src/activities.ts:214]: Do we need this commented out?

Comment (docirl) [reply examples/odata-cli/src/activities.ts:214]: This is used for manually testing, so I've added this blocked commented as it should not be run together with business object search and can be uncommented and used for testing when needed.

**Review by devinea — APPROVED** [review]
Code changes are clear and covered by tests. 
I haven't tested locally. 
changeset ✅

File: examples/odata-cli/src/activities.ts (hunk 1 above)
Comment (IainSAP) [reply examples/odata-cli/src/activities.ts:214]: Can you add a comment to indicate thats why this is here?

**Review by IainSAP — COMMENTED** [review]
One minor comment, not familiar with the functionality here so can only review from a code perspective.

**Comment by korotkovao** [comment]
> One minor comment, not familiar with the functionality here so can only review from a code perspective.

Thank you for the review, I have edited the comment

**Review by IainSAP — APPROVED** [review]
Thanks

- Not tested manually
- Comment addressed

**Review by devinea — APPROVED** [review]
re-approving

**Comment by korotkovao** [comment]
Thank you @devinea @IainSAP

**Pull request merged**